### PR TITLE
Cleanup after merging wnode-status and statusd

### PR DESCRIPTION
**Pull request by divan**
This PR cleans up tests and references to wnode-status after https://github.com/status-im/status-go/pull/550

### CODE REVIEW

**Review by themue — APPROVED** [review]
Had to made a connection between "shh" and Whisper first. Again learned something new.

LGTM

File: static/tests/mailserver.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('Whisper MailServer', () => {
 
     describe('Check prerequisites', () => {
         console.log('Expecting MailServer running.')
-        console.log('./build/bin/wnode-status -mailserver -passwordfile=./static/keys/wnodepassword -http -httpport 8540 -listenaddr=127.0.0.1:8549 -identity=./static/keys/wnodekey')
+        console.log('./build/bin/statusd -les=false -shh -shh.mailserver -passwordfile=./static/keys/wnodepassword -http -httpport 8540 -listenaddr=127.0.0.1:8549 -identity=./static/keys/wnodekey')
```
Comment (adambabik) [inline static/tests/mailserver.js:16]: So, `les` is `true` by default?

**Review by adambabik — APPROVED** [review]

File: static/tests/mailserver.js (hunk 1 above)
Comment (divan) [reply static/tests/mailserver.js:16]: Uff, I assumed so. But now I see that it's `false`. I think it should be enabled as LES node is what most of `statusd` usages probably will be, what do you think?

Comment (themue) [reply static/tests/mailserver.js:16]: Yes, would vote for it as it hopefully will become a standard protocol for most clients anywhere.

Comment (adambabik) [reply static/tests/mailserver.js:16]: I would keep it as `false` by default actually. By default I expect to start as little protocols as possible and opt-in for the ones I need.

Comment (divan) [reply static/tests/mailserver.js:16]: @adambabik sorry, just saw your comment. Default behaviour for `statusd` is to run les. But we can change it, of course.

**Pull request merged**